core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.ts

```diff
@@ -574,6 +577,43 @@ export class StateTranslationComponent
     return translatableContents;
   }
 
+  getActiveTab(): string {
+    this.initActiveContentId = this.stateEditorService.
+      getInitActiveContentId();
+    if (!this.initActiveContentId) {
+      return null;
+    }
+    const tabName = this.stateEditorService.getInitActiveContentId().
+      split('_')[0];
+    return tabName === 'default' ? this.TAB_ID_FEEDBACK : tabName;
```

Comment:
Can this constant be picked up from a definition instead of being defined here adhoc?

Reply:
Umm, I don't think it's required thou since the logic is quite simple and also we are using this for this particular case, WDYT ?